c/models/dt_ftrl_real.cc

```diff
@@ -388,20 +388,19 @@ double FtrlReal<T>::fit(T(*linkfn)(T), T(*lossfn)(T,U)) {
               }
             }
           }
-          #pragma omp atomic
           loss += loss_local;
```

Comment:
With your implementation of `dt::atomic`, will it do `loss += loss_local` atomically or we need to use some `loss.operation()` here?